geometry_msgs/PoseStamped goal_pose
duration time_from_start
---
bool success